config/custom.css

```diff
@@ -0,0 +1,185 @@
+      }
+ 
+      .mainmenufooter a {
+      color: #026C96;
+      font-family: helvetica,arial,sans-serif;
+      }
+ 
+      .mainmenufooter a: hover {
+      color: #0083BC;
+      }
+ 
+      .label {
+      font-family: helvetica,arial,sans-serif;
+      }
+ 
+      .maintabbarbottom {
+      background: #FFF;
+      border: 1px solid #000;
+      }
+ 
+      .tabbar a.button.cur, .tabbar a.button.cur: hover {
+      color: #000;
+      background: #FFF;
+      border: 1px solid #000;
+      font-family: helvetica,arial,sans-serif;
+      text-shadow: none;
+      }
+ 
+      .tabbar a.button {
+      color: #000;
+      background: #D1CAFF;
+      border: 1px solid #000;
+      font-family: helvetica,arial,sans-serif;
+      text-shadow: none;
+      }
+ 
+      .tabbar a.button: hover {
+      color: #000;
+      background: #E1DCFF;
+      border: 1px solid #000;
+      font-family: helvetica,arial,sans-serif;
+      text-shadow: none;
+      }
+ 
+ 
+      .broadcast-red {
+      background: none repeat scroll 0% 0% #EC5C4D;
+      color: #000;
+      border: 2px solid #000;
+      padding: 5px;
+      font-family: helvetica,arial,sans-serif;
+      font-size: 10pt;
+      }
+ 
+      .broadcast-blue {
+      background: none repeat scroll 0% 0% #8FA6B3;
+      color: #000;
+      border: 2px solid #000;
+      padding: 5px;
+      font-family: helvetica,arial,sans-serif;
+      font-size: 10pt;
+      }
+ 
+      .broadcast-green {
+      background: none repeat scroll 0% 0% #9BB38F;
+      color: #000;
+      border: 2px solid #000;
+      padding: 5px;
+      font-family: helvetica,arial,sans-serif;
+      font-size: 10pt;
+      }
+ 
+      .pm-window h3 {
+      font-family: helvetica,arial,sans-serif;
+      color: #212121;
+      border: 1px solid #212121;
+      background: #FFF;
+      }
+   
+      .pm-window h3: hover {
+      color: #000;
+      background: #FFF;
+      }
+ 
+      .pm-window.focused h3, .pm-window.focused h3: hover {
+      color: #000;
+      background: #FFF;
+      }
+ 
+      .pm-log, .chat-log, .battle-log, .ps-popup {
+      font-family: helvetica,arial,sans-serif;
+      background: url(http://i.imgur.com/Q8vHT0Y.png);
+      }
+ 
+      .chat-log {
+      border: 1px solid #757575;
+      }
+ 
+      .pm-log-add {
+      background: rgba(31, 35, 91, 0.8);
+      }
+ 
+      .userlist, .chat-log-add {
+      background: #E1DCFF;
+      border-right: 1px solid #E1DCFF;
+      border-top: 1px solid #E1DCFF;
+      }
+ 
+      .closebutton, .minimizebutton {
+      color: #9484FF;
+      }
+ 
+      .chatbox textarea {
+      font-family: helvetica,arial,sans-serif;
+      }
+ 
+      .ps-popup .label {
+      color: #000;
+      }
+ 
+      .label strong {
+      color: #000;
+      }
+ 
+      .ps-popup p.error {
+      color: red;
+      }
+ 
+      .ps-overlay {
+      background: rgba(31, 35, 91, 0.5);
+      }
+ 
+      a.ilink {
+      color: #026C96;
+      }
+ 
+      a.ilink: hover {
```

Comment:
No spaces after colons when it is a selector (that is, when it is in front of an opening brace)